Strategies/protocols/psyfi/actions/deposit.ts

```diff
@@ -80,7 +138,7 @@ export const deposit = async (
     }
 
     // Create transfer to deposit receipt instruction
-    coreDepositInstruction = await psyFiInstructions.transferToDepositReceiptInstruction(
+    tokenInstruction = await psyFiInstructions.transferToDepositReceiptInstruction(
```

Comment:
naming this `tokenInstruction` doesn't seem to capture the intention of this instruction. why not name it as e.g. transferToReceiptIx

Reply:
same for `additionalTokenInstructions`